flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/utils/ExpressionTestBase.scala

```diff
@@ -77,7 +77,7 @@ abstract class ExpressionTestBase {
   protected val notNullable = "not null"
 
   // used for accurate exception information checking.
-  val expectedException: ExpectedException = ExpectedException.none()
+  var expectedException: ExpectedException = ExpectedException.none()
```

Comment:
do we still need to change this to `var`?

Reply:
> do we still need to change this to `var`?

I have changed it.